--- swfdec-0.9.2/swfdec/swfdec_as_native_function.c~	2008-11-06 23:24:03.000000000 +0100
+++ swfdec-0.9.2/swfdec/swfdec_as_native_function.c	2015-05-10 02:04:28.648375664 +0200
@@ -230,7 +230,8 @@
  * that takes the value. For the conversion, the default conversion functions 
  * like swfdec_as_value_to_string() are used. If not enough arguments are 
  * available, the function stops converting and returns %NULL. The following 
- * conversion characters are allowed:<itemizedlist>
+ * conversion characters are allowed:
+ * <itemizedlist>
  * <listitem><para>"b": convert to boolean. Requires a %gboolean pointer
  *                 </para></listitem>
  * <listitem><para>"i": convert to integer. Requires an %integer pointer
@@ -256,7 +257,7 @@
  *		   initialized to the empty value for their type. This 
  *		   conversion character is only allowed once in the conversion 
  *		   string.</para></listitem>
- * </itemizedlist>
+ *</itemizedlist>
  *
  * Returns: %TRUE if the conversion succeeded, %FALSE otherwise
  **/
--- swfdec-0.9.2/swfdec/swfdec_player.c~	2008-11-06 23:24:03.000000000 +0100
+++ swfdec-0.9.2/swfdec/swfdec_player.c	2015-05-10 02:05:11.415602069 +0200
@@ -2322,7 +2322,8 @@
    * communicate with the hosting environment. In web browsers it is used to 
    * call Javascript functions. Standalone Flash players understand a limited 
    * set of functions. They vary from player to player, but the most common are 
-   * listed here: <itemizedlist>
+   * listed here:
+   * <itemizedlist>
    * <listitem><para>"quit": quits the player.</para></listitem>
    * <listitem><para>"fullscreen": A boolean setting (parameter is "true" or 
    * "false") that sets the player into fullscreen mode.</para></listitem>
