diff -urN swfdec-0.1.3.orig/configure.ac swfdec-0.1.3/configure.ac
--- swfdec-0.1.3.orig/configure.ac	Wed Jan  8 20:29:17 2003
+++ swfdec-0.1.3/configure.ac	Wed Jan  8 20:32:38 2003
@@ -21,6 +21,7 @@
 AM_MAINTAINER_MODE
 
 AC_PROG_CC
+AC_PROG_LIBTOOL
 AM_PROG_CC_STDC
 AM_PROG_AS
 AS="${CC}"
@@ -76,16 +77,9 @@
 fi
 
 MOZILLA_CFLAGS=
-if test -z "$MOZILLA_CONFIG"; then
-  AC_PATH_PROG(MOZILLA_CONFIG, mozilla-config, no)
-fi
-if test "x$MOZILLA_CONFIG" != "xno"; then
-  _mozilla_include_dir=`mozilla-config --cflags|sed 's/-I\(.*\) .*/\1/'`
-  MOZILLA_CFLAGS="-I$_mozilla_include_dir/plugin -I$_mozilla_include_dir/java -I$_mozilla_include_dir/nspr"
-else
-  AC_MSG_WARN([mozilla-config not found. Mozilla/Netscape plugin will not be built])
-fi
-AM_CONDITIONAL(WITH_MOZILLA,[test "$MOZILLA_CFLAGS" != ""])
+_mozilla_include_dir="`$PKG_CONFIG --variable=includedir mozilla-gtkmozembed`"
+MOZILLA_CFLAGS="-I$_mozilla_include_dir/plugin -I$_mozilla_include_dir/java -I`$PKG_CONFIG --variable=includedir mozilla-nspr`/nspr"
+AM_CONDITIONAL(WITH_MOZILLA,[test "$MOZILLA_CFLAGS" != ""])
 AC_SUBST(MOZILLA_CFLAGS)
 
 dnl GdkPixbuf loader
