--- swfdec-0.3.4/configure.ac.orig	2005-03-20 05:56:29.000000000 +0100
+++ swfdec-0.3.4/configure.ac	2005-03-21 22:05:50.295326376 +0100
@@ -132,15 +132,8 @@
 fi
 
 MOZILLA_CFLAGS=
-if test -z "$MOZILLA_CONFIG"; then
-  AC_PATH_PROG(MOZILLA_CONFIG, mozilla-config, no)
-fi
-if test "x$MOZILLA_CONFIG" != "xno"; then
-  _mozilla_include_dir=`$MOZILLA_CONFIG --cflags|sed 's/-I\(.*\) .*/\1/'`
-  MOZILLA_CFLAGS="-I$_mozilla_include_dir/plugin -I$_mozilla_include_dir/java -I$_mozilla_include_dir/nspr"
-else
-  AC_MSG_WARN([mozilla-config not found. Mozilla/Netscape plugin will not be built])
-fi
+  _mozilla_include_dir="`$PKG_CONFIG --variable=includedir mozilla-gtkmozembed`"
+  MOZILLA_CFLAGS="-I$_mozilla_include_dir/plugin -I$_mozilla_include_dir/java -I`$PKG_CONFIG --variable=includedir mozilla-nspr`/nspr"
 AM_CONDITIONAL(WITH_MOZILLA,[test "$MOZILLA_CFLAGS" != ""])
 AC_SUBST(MOZILLA_CFLAGS)
 
